lib/core/src/model.rs

```diff
@@ -1434,7 +1443,18 @@ impl Payment {
             swapper_fees_sat: Some(swap.swapper_fees_sat),
             payment_type,
             status: swap.status,
-            details: payment_details,
+            details: PaymentDetails::Lightning {
```

Comment:
I don't think the `from_pending_swap()` handling should be reverted. It should handle the Bitcoin payment_details when the receive chain swap becomes pending without a tx

Reply:
My bad Ross, must've overlooked it during rebase, will fix :+1: